zanata-war/src/main/java/org/zanata/service/impl/DocumentServiceImpl.java

```diff
@@ -265,8 +265,10 @@ private void processWebHookDocumentMilestoneEvent(
             WordStatistic oldStats = StatisticsUtil.copyWordStatistic(stats);
             if(oldStats != null) {
 
-                for(Map.Entry<ContentState, Integer> entry: event.getWordDeltasByState().entrySet()) {
-                    oldStats.increment(entry.getKey(), entry.getValue());
+                for (Map.Entry<ContentState, Long> entry : event
+                        .getWordDeltasByState().entrySet()) {
+                    oldStats.increment(entry.getKey(),
+                            entry.getValue().intValue());
```

Comment:
I mean, ideally we would use long/Long consistently, instead of downgrading long to int anywhere. But if we're not going to do that, we should at least have a comment making it clear that we are "casting" long to int here.